extensions/interactions/NumericExpressionInput/directives/numeric-expression-input-interaction.component.html

```diff
@@ -3,3 +3,26 @@
 <div ng-show="!$ctrl.isCurrentAnswerValid()" class="alert alert-danger mt-2">
   <[$ctrl.warningText]>
 </div>
+<!--
+  Forced to use !important here as it makes the code easy to understand, but does not align with design principles.
+  Normal CSS styles aren't working as Guppy styles overrides them by default. Only way to fix this is to override the
+  specificity using !important.
+-->
```

Comment:
You explain why you use important but not why is this needed at all. Explain what was the issue before this code was added.

Reply:
I'll try to explain in a better way within the comments.

Reply:
Done!